### PR TITLE
Add Fluidity: Spin Cycle

### PR DESCRIPTION
**Pull request by Scotsguy**
Adds Fluidity: Spin Cycle.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/fluidity-spin-cycle/game.dat [hunk 1]
```diff
@@ -0,0 +1,10 @@
+title = "Fluidity: Spin Cycle"
+description = "Fluidity: Spin Cycle, known as Hydroventure: Spin Cycle in Europe and Oceania, is a physics puzzle game developed by Curve Studios and published by Nintendo for the Nintendo 3DS eShop. The game is a sequel to the WiiWare game Fluidity, the latter of which focuses on controlling a small body of water that is utilized in its different states of matter to progress through the game's levels. The game is based around gyroscope controls and tilting the 3DS to move."
```
Comment (CodingKoopa) [inline games/fluidity-spin-cycle/game.dat]: " for the Nintendo 3DS eShop" is uneeded.

File: games/fluidity-spin-cycle/game.dat [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+needs_system_files = false
+needs_shared_font = false
+game_type = "eshop"
+
+[[releases]]
+title = "00040000000AD300"
+region = "eur"
```
Comment (CodingKoopa) [inline games/fluidity-spin-cycle/game.dat]: Add a `usa` release.

**Comment by Scotsguy** [comment]
I don't have the USA version, but I did add the ID and removed the redundant eShop part

**Comment by Leo626** [comment]
> I don't have the USA version, but I did add the ID and removed the redundant eShop part

That is fine then but USA is preferred so someone will want to change later.

**Pull request merged**